CryptoPkg/Library/BaseCryptLib/Rand/CryptRand.c

```diff
@@ -49,7 +47,11 @@ RandomSeed (
   if (Seed != NULL) {
     RAND_seed (Seed, (UINT32)SeedSize);
   } else {
-    RAND_seed (DefaultSeed, sizeof (DefaultSeed));
+    if (!GetRandomNumber32 (&RandomNumber)) {
+      return FALSE;
+    }

```

Comment:
Will this cause compatibility problem?

What if the random pool does not have enough entropy at that time?